app/main.py

```diff
@@ -1,8 +1,61 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: float,
+                 clean_mark: float,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: float, average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+        self.rating_list = [average_rating for i in range(count_of_ratings)]
+
+    def serve_cars(
+            self,
+            car_list: [Car]
+    ) -> float:
+        total_cost = 0
+        for car in car_list:
+            wash_cost = self.calculate_washing_price(car)
+            if self.wash_single_car(car):
+                total_cost += wash_cost
+        return total_cost
+
+    def calculate_washing_price(
+            self,
+            car: Car
+    ) -> float:
+        return round(car.comfort_class
+                     * (self.clean_power - car.clean_mark)
+                     * (self.average_rating / self.distance_from_city_center),
+                     1)
+
+    def wash_single_car(
+            self,
+            car: Car
+    ) -> bool:
```

Comment:
```suggestion
    def wash_single_car(self, car: Car) -> bool:
```

Reply:
Polyakiv Andrii told me to place parameters in different rows, should I have ignored him?
Please give me a clear response so I can understand what different mentors want from me (because otherwise I will make the same mistakes all the time)
Ofc it's not a must, so feel free to ignore